ext/CatalystCairoMakieExtension/cairo_makie_extension_spatial_modelling.jl

```diff
@@ -100,8 +101,10 @@ function lattice_animation(
     x_vals, y_vals = Catalyst.extract_grid_axes(lrs)
 
     # Creates the base figure (which is modified in the animation).
-    fig, ax, hm = heatmap(x_vals, y_vals, vals[1];
-        axis = (xgridvisible = false, ygridvisible = false, xlabel = "Compartment", ylabel = "Compartment"),
+    fig, ax,
+    hm = heatmap(x_vals, y_vals, vals[1];
+        axis = (xgridvisible = false, ygridvisible = false,
+            xlabel = "Compartment", ylabel = "Compartment"),
```

Comment:
Things likes this are not a reasonable reformat, and are why Catalyst no longer uses JuliaFormatter. Please revert this PR. Is this even correct syntactically now?